src/IW/Effects/Cabal.hs

```diff
@@ -63,13 +62,7 @@ repoCabalUrl (RepoOwner repoOwner) (RepoName repoName) = Url $
 
 -- | Parses a comma separated @Text@ value to @[Category]@.
 categoryNames :: GenericPackageDescription -> [Category]
-categoryNames genPkgDescr = Category . strip <$> splitCategories genPkgDescr
+categoryNames genPkgDescr = Category <$> splitCategories genPkgDescr
   where
     splitCategories :: GenericPackageDescription -> [Text]
-    splitCategories = splitOnNonEmptyText "," . toText . category . packageDescription
-
-    -- | A variation of @splitOn@ that returns @[]@ instead of @[""]@
-    -- if the text to be split is empty.
-    splitOnNonEmptyText :: Text -> Text -> [Text]
-    splitOnNonEmptyText _ ""       = []
-    splitOnNonEmptyText delim text = splitOn delim text
+    splitCategories = splitAndStrip "," . toText . category . packageDescription
```

Comment:
If I recall correctly, `splitAntStrip` function is now in `Prelude`. I propose to move it in this module. It's not going to be used in many modules (probably it's going to be used only here), so no need to move into `Prelude`. Candidates for adding to `Prelude` are functions or types that used in at least half of the project and unambiguous.

Reply:
@chshersh I agree